common/scripted_triggers/wc_war_triggers.txt

```diff
@@ -1,7 +1,20 @@
-﻿has_sea_invader_innovation = {
+﻿culture_can_raid_at_sea_even_if_feudal_trigger = {
 	OR = {
 		has_innovation = innovation_longboats
-		has_cultural_parameter = culture_can_raid_at_sea_even_if_feudal
+		prefers_piracy_trigger = yes
+		
+		AND = {
+			# Enables sea raiding for tradition_practiced_pirates
+			has_cultural_parameter = culture_can_raid_at_sea_even_if_feudal
+			OR = {
+				# Cultures not included can only raid on land
+				is_culture_or_parent_culture_trigger = { CULTURE = culture:kvaldir }
+				is_culture_or_parent_culture_trigger = { CULTURE = culture:saurok }
+				is_culture_or_parent_culture_trigger = { CULTURE = culture:black_dragon }
+				is_culture_or_parent_culture_trigger = { CULTURE = culture:northsea }
+				is_culture_or_parent_culture_trigger = { CULTURE = culture:stormreaver }
+			}
+		}
```

Comment:
You blocked the cultures who picked `tradition_practiced_pirates` from sea raiding.
For example, Stormwind picks `tradition_practiced_pirates`, they have `culture_can_raid_at_sea_even_if_feudal` from the tradition thus it returns true, The OR condition returns false, and thus AND returns false.

Reply:
This is intentional. Only the listed cultures have feudal sea raiding with `tradition_practiced_pirates` as either a function of their anatomy or core cultural identity, Everyone else who picks this culture can also gain feudal raiding, but cannot do it across sea tiles.
See `00_scripted_rule_triggers.txt`:
```
can_raid_trigger = {
	OR = {
		government_has_flag = government_can_raid_rule
		faith = { has_doctrine_parameter = faith_can_raid }
		AND = {
			exists = dynasty
			dynasty = { has_dynasty_modifier = fp1_legacy_of_piracy_modifier }
		}
		
		# Warcraft - Allows feudal raiding with this tradition
		culture = { has_cultural_tradition = tradition_practiced_pirates }
	}
	NOT = {
		faith = {
			has_doctrine_parameter = holy_wars_forbidden
		}
	}
}
```